tests/test_iqm_sampler.py

```diff
@@ -220,6 +224,8 @@ def test_run_sweep_has_heralding_mode_none_by_default(
     run_result = RunResult(status=Status.READY, measurements=[{'some stuff': [[0], [1]]}], metadata=iqm_metadata)
     kwargs = create_run_request_default_kwargs
     assert sampler._compiler_options.heralding_mode == HeraldingMode.NONE
+    kwargs = create_run_request_default_kwargs
```

Comment:
Either a diff artifact, or 2 identical lines added
![Screenshot 2024-10-16 at 9 38 45](https://github.com/user-attachments/assets/78ad4cc8-01c7-49d8-8522-519ff1f89315)

Reply:
The same on lines 249 and 257